sdk/formrecognizer/ai-form-recognizer/src/options/FormRecognizerClientOptions.ts

```diff
@@ -68,6 +68,17 @@ export interface FormRecognizerCommonClientOptions extends CommonClientOptions {
    * Default: FormRecognizerApiVersion.Latest ("2021-09-30-preview")
    */
   apiVersion?: FormRecognizerApiVersion;
+  /**
+   * Gets or sets the audience to use for authentication with Azure Active Directory.
```

Comment:
gets/sets feels out of place for a property of an object that is consumed by the client constructor. Setting it after the fact won't change the scope used, right?

The rest of this doc string is great, but for the first line, how about just:

"The audience (scope) to use for authentication with Azure Active Directory."

Reply:
> gets/sets feels out of place for a property of an object that is consumed by the client constructor. 

Copied this from some other package.